docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3.7"
+services:
+
+  thrift:
+    build: docker/thrift
```

Comment:
I would be interested in using @NielsZeilemaker his suggestion and using a pre-existing image instead of building one from scratch.

Reply:
We're also open for PR's :-)

Reply:
oh of course, didn't know there was something out there